clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -62,7 +64,7 @@ const ConsentBanner: FunctionComponent<BannerProps> = ({
       className={`fides-banner 
         fides-banner-bottom 
         ${bannerIsOpen ? "" : "fides-banner-hidden"} 
-        ${window.Fides.options.tcfEnabled ? "fides-tcf-banner-container" : ""}`}
+        ${className}`}
```

Comment:
it's a little cleaner to pass in a conditional `className` rather than `isTcf`—and more extensible/adaptable!